microsoft-edge/web-platform/release-notes/139.md

```diff
@@ -0,0 +1,560 @@
+
+CSS transitions that are set to `none`, while running, now continue to run.  When transition-related CSS properties change, those changes only affect newly started transitions.
+
+This change makes Microsoft Edge consistent with other browsers.  Previously, Microsoft Edge incorrectly canceled transitions when the `transition` property was set to `none`.
+
+See also:
+* [transition](https://developer.mozilla.org/docs/Web/CSS/transition) at MDN.
+
+
+<!-- ---------- -->
+###### CSS `corner-shape` property
+
+The `corner-shape` CSS property allows you to specify the shape of the corners of an element, in addition to the existing `border-radius` property.
+
+This allows to create shapes like _squircles_ (squared-off circles) or notches, and animating between them.
+
+See also:
+* [Corner Shaping: the corner-shape property](https://drafts.csswg.org/css-borders-4/#corner-shaping) in _CSS Borders and Box Decorations Module Level 4_.
+
+
+<!-- ---------- -->
+###### Support `width` and `height` presentational attributes on nested `<svg>` elements
+
+You can now use the `width` and `height` presentational attributes on nested `<svg>` elements, through both SVG markup and CSS.  This approach provides greater flexibility, allowing you to style SVG elements more efficiently within complex designs.
+
+With this feature, the following two HTML code snippets now produce the same output:
+
+```html
+<svg width="100px" height="100px">
+  <svg style="width:50px;height:50px;">
+    <circle cx="50px" cy="50px" r="40px" fill="green" />
+  </svg>
+</svg>
+```
+
+```html
+<svg width="100px" height="100px">
+  <svg width="50px" height="50px">
+    <circle cx="50px" cy="50px" r="40px" fill="green" />
+  </svg>
+</svg>
+```
+
+In the second line:
+* The first example uses a `style` attribute that contains width and height values.
+* The second example uses separate `width` and `height` attributes.
+
+See also:
+* [`<svg>`](https://developer.mozilla.org/docs/Web/SVG/Reference/Element/svg) at MDN.
+
+
+<!-- ------------------------------ -->
+#### Web APIs
+
+
+<!-- ---------- -->
+###### `request-close` invoker command
+
+The `<dialog>` element's `requestClose()` method can now also be invoked declaratively (as well as directly), by using the `request-close` invoker command.
+
+The `requestClose()` method and the `request-close` invoker command are used to request closing a dialog element, which fires an event that allows you (the developer) to prevent the dialog from closing, if necessary.
+
+See also:
+* [Invoker Commands API](https://developer.mozilla.org/docs/Web/API/Invoker_Commands_API) at MDN.
+
+
+<!-- ---------- -->
+###### Allow more characters in JavaScript DOM APIs
+
+You can now create DOM elements and attributes with a wider variety of valid characters and names<!-- todo: whose names contain a wider variety of valid characters --> in JavaScript, matching the behavior for when the same elements and attributes are created in HTML markup.
```

Comment:
clarify - 
* characters in names
* characters and names
* characters and thus names

@captainbrosset